test/src/SiteClient_test.js

```diff
@@ -185,6 +194,9 @@ describe('Site API', () => {
       vcr(async () => {
         const roles = await client.roles.all();
 
+        console.log(`To make test pass, update plan for this site to 100.`);
+        console.log(`Site ID -> ${site.id}`);
```

Comment:
tests were passing here, can we delete this?

Reply:
This applies only if you have destroyed the cassette. Every time I delete the cassette and that test fails I get lost, so I put a reminder.